docker-compose.yaml

```
@@ -0,0 +1,17 @@
+services:
+  mc-server:
+    build: .
```

@A-Marbach as i said you can only download the image on docker build and here is an example how you can set the `SERVER_JAR_URL` ARG for Dockerfile via your Compose File.

```suggestion
    build: .
      args:
        SERVER_JAR_URL:${SERVER_JAR_URL} # needs to be set in .env and in Dockerfile as ARG SERVER_JAR_URL=your_default_url
```

---

Thanks, updated docker-compose.yml to pass SERVER_JAR_URL ARG from .env to Dockerfile as suggested.